extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,318 @@
+# KHR_xmp
+
+## Contributors
+
+TODO
```

Comment:
We should add names here before merging.
@emilian0 
@robertlong 
@donmccurdy 
@andreasplesch 
@lexaknyazev 
@emackey 

Did I miss anyone?